Makefile-lib.am

```diff
@@ -44,7 +44,7 @@ RpmOstree_1_0_gir_INCLUDES = OSTree-1.0 Gio-2.0
 RpmOstree_1_0_gir_CFLAGS = $(librpmostree_1_la_CFLAGS)
 RpmOstree_1_0_gir_LIBS = librpmostree-1.la
 RpmOstree_1_0_gir_SCANNERFLAGS = --warn-all --identifier-prefix=RpmOstree --symbol-prefix=rpm_ostree
-RpmOstree_1_0_gir_FILES = $(librpmostreeinclude_HEADERS) $(filter-out %-private.h,$(librpmostree_1_la_SOURCES))
+RpmOstree_1_0_gir_FILES = $(librpmostreeinclude_HEADERS) $(filter-out %-private.c,$(librpmostree_1_la_SOURCES))
```

Comment:
A bit more future proof to filter out `%-private.[ch]` i.e. do both.

Reply:
Is that the makefile syntax for that? I tried with `%-private.%` earlier but it didn't seem to work. I'll retry your way.

Reply:
I didn't manage to get that syntax to work, I don't think it's a valid makefile one. Tried again the double-% wildcard too, but didn't work either. This now filters .c and .h explicitly.

Reply:
Yeah, sorry about that invalid suggestion.  I think there can only be one `%`.